frontend/packages/kubevirt-plugin/src/utils/strings.ts

```diff
@@ -18,6 +18,16 @@ export const ADD_NETWORK_INTERFACE = 'Add Network Interface';
 export const ADD_SNAPSHOT = 'Take Snapshot';
 export const RESTORE_SNAPSHOT = 'Restore Snapshot';
 
+// provision sources help text
+export const PROVISION_SOURCE_PXE_HELPTEXT =
+  'Boot an operating system from a server on a network. This will show up as a NIC on the Networking step. In order to PXE boot, you need to have a Network Attachment Definition available for you either in this namespace, or in the default one';
+export const PROVISION_SOURCE_CONTAINER_HELPTEXT =
+  'Link to a bootable operating system container located in a registry accessible from the cluster. Example: kubevirt/cirrosregistry-disk-demo. This action will not create a persistent volume claim (PVC). This will show up as a disk in the Storage step';
```

Comment:
So, we need to highlight the limitations of this and reference fedora instead of cirros:
Link to a bootable operating system container located in a registry accessible from the cluster. Example: kubevirt/fedora-cloud-container-disk-demo. This action will not create a persistent volume claim (PVC). This will show up as a disk in the Storage step. The container disk is meant to be used only for read-only filesystems such
as CD-ROMs or for small short-lived throw-away VMs.';

And then, once picked, under it it needs to show the example kubevirt/fedora-cloud-container-disk-demo so the user can actually copy paste it to the text box.